### PR TITLE
Added veeam unofficial

### PR DESCRIPTION
**Pull request by jorgedlcruz**
A collection of unofficial icons inspired from the Veeam Visio shapes we all know and like.

### CODE REVIEW

**Comment by ad1992** [comment]
@jorgedlcruz can you add a `png` image instead of `jpg` ?, the image is breaking in the preview

**Comment by jorgedlcruz** [comment]
Added the .png as requested @ad1992

File: libraries.json [hunk 1]
```diff
@@ -870,5 +870,20 @@
...
+    "authors": [
+      {
+        "name": "Jorge de la Cruz",
+        "url": "https://www.jorgedelacruz.uk"
+      }
+    ],
+    "source": "jdelacruz/veeam_unofficial.excalidrawlib",
+    "preview": "jdelacruz/veeam_unofficial.png",
+    "created": "2021-09-16",
+    "updated": "2021-09-16"
```
Comment (dwelle) [inline libraries.json:886]:
```suggestion
    "created": "2021-09-21",
    "updated": "2021-09-21"
```

**Comment by dwelle** [comment]
Looking good. Thanks!

**Pull request merged**